general/development/process.md

```diff
@@ -101,10 +103,26 @@ Whenever a solution for an issue is finished, it is submitted to the standard in
 
 ## Major release cycles
 
-Since Moodle 2.0, we have a policy of release major versions (eg 2.1, 2.2) every six months in May and November. See the [General release calendar](../releases.md#general-release-calendar) for more details.
+Moodle release versions are in the format x.y.z where the x.y is a major release, and the z is a minor release.
+
+<Since version="2.0"/>
+
+Since Moodle 2.0, we have a policy of release major versions (for example 4.0, 4.1) every six months in May and November.
```

Comment:
It might be good to amend this sentence  as well, to say "Between Moodle 2.0 and Moodle 4.1, we had a policy ....",